### PR TITLE
Mmi 5676 slack channel qa investigate missing products and pools

### PR DESCRIPTION
**Pull request by jpsains**

### CODE REVIEW

File: packages/adapters-library/src/core/adaptersController.ts [hunk 1]
```diff
@@ -299,6 +301,22 @@ export class AdaptersController {
...
+            .getProtocolTokens()
+            .catch(() => undefined)
+
+          if (protocolTokens) {
+            if (!product.protocolTokens) {
+              product.protocolTokens = {}
+            }
+
+            product.protocolTokens[chainId] = protocolTokens
+          }
```
Comment (jpsains) [inline packages/adapters-library/src/core/adaptersController.ts:319]: Now can see full protocol tokens with underlying when running support with flag -pt

File: packages/adapters-library/src/scripts/checksumAddress.ts
Comment (bergarces) [inline packages/adapters-library/src/scripts/checksumAddress.ts:1]: This should be under `core/decorators`.

File: packages/adapters-library/src/defiProvider.ts [hunk 2]
```diff
@@ -873,6 +883,7 @@ export class DefiProvider {
   async getSupport(input?: {
     filterChainIds?: Chain[] | undefined
     filterProtocolIds?: Protocol[] | undefined
+    includeProtocolTokens?: boolean
```
Comment (bergarces) [inline packages/adapters-library/src/defiProvider.ts:886]: With this new parameter added. Shouldn't it also use the new decorator?

File: packages/adapters-library/src/scripts/checksumAddress.ts
Comment (jpsains) [reply packages/adapters-library/src/scripts/checksumAddress.ts:1]: Thanks, was in process of moving to utils but decorators much better

File: packages/adapters-library/src/defiProvider.ts (hunk 2 above)
Comment (jpsains) [reply packages/adapters-library/src/defiProvider.ts:886]: nope......this is a boolean to include protocol tokens it is not a address input

**Pull request merged**